domain/tshirt.js

```diff
@@ -10,19 +10,26 @@ function invalidData(body) {
   return _.isUndefined(body.size) || _.isUndefined(body.model);
 }
 
+tshirt.NoTShirt = function () {
+  return {
+    details: [{ size: "L", model: "Slim fit" }],
```

Comment:
The motivation behind the default values is to simplify the views. In the admin edit participant view there are elements that display tshirt info, even when these elements will end up hidden from the user, in the server side jade tries to fill these values  with information from the participant.